### PR TITLE
fix(discovery): Throw error on wrong path

### PR DESCRIPTION
**Pull request by shubham-cmyk**
## Explanation

Throw a error if not exist path is provided.

## Related issue

Fixes https://github.com/kyverno/chainsaw/issues/805

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
- [x] I have read the [PR documentation guide](https://github.com/kyverno/kyverno/blob/main/.github/pr_documentation.md) and followed the process including adding proof manifests to this PR.
- [x] This is a bug fix and I have added unit tests that prove my fix is effective.

### CODE REVIEW

File: pkg/runner/.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (eddycharly) [inline pkg/runner/.json:1]: what's that ?

Comment (shubham-cmyk) [reply pkg/runner/.json]: a null test report generated by mistake

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,5 @@ chainsaw
 coverage.out
 website/site
 pkg/runner/chainsaw.xml
+**/.json
```
Comment (eddycharly) [inline .gitignore:8]: please exclude only specific files

File: pkg/commands/migrate/kuttl/tests/command_test.go [hunk 3]
```diff
@@ -7,6 +7,7 @@ import (
 	"path/filepath"
 	"testing"
 
+	mockName "github.com/dustinkirkland/golang-petname"
```
Comment (eddycharly) [inline pkg/commands/migrate/kuttl/tests/command_test.go:10]:
```suggestion
	petname "github.com/dustinkirkland/golang-petname"
```

File: pkg/utils/fs/discover.go [hunk 4]
```diff
@@ -1,30 +1,41 @@
...
 			if err != nil {
-				return nil, err
+				return err
+			}
+			if info.IsDir() {
+				folders.Insert(file)
 			}
+			return nil
+		})
+
```
Comment (eddycharly) [inline pkg/utils/fs/discover.go:32]: remove empty lines ;)

File: .gitignore (hunk 2 above)
Comment (shubham-cmyk) [reply .gitignore:8]: done

**Pull request merged**